account_operating_unit/tests/test_cross_ou_journal_entry.py

```diff
@@ -93,37 +94,42 @@ def test_cross_ou_journal_entry(self):
         move_vals.update(
             {"journal_id": journal_ids and journal_ids.id, "line_ids": lines}
         )
-        move = self.move_model.with_user(self.user_id.id).create(move_vals)
+        move = (
+            self.move_model.with_user(self.user_id.id)
+            .with_context(check_move_validity=False)
+            .create(move_vals)
+        )
         # Post journal entries
         move.action_post()
         # Check the balance of the account
         self._check_balance(self.current_asset_account_id.id, acc_type="other")
         clearing_account_id = self.company.inter_ou_clearing_account_id.id
         self._check_balance(clearing_account_id, acc_type="clearing")
         # Report journal
-        report_journal = (
-            self.env["report.account.report_journal"]
-            .sudo()
-            ._get_report_values(
-                docids=[journal_ids.id],
-                data={
-                    "form": {
-                        "journal_ids": journal_ids.ids,
-                        "company_id": journal_ids.company_id,
-                        "used_context": {
-                            "operating_unit_ids": journal_ids.operating_unit_id.id
-                        },
-                    }
-                },
-            )
-        )
-        self.assertTrue(report_journal)
+        # Commented as this report does not longer exist
```

Comment:
if there is no equivalent report just remove this completely

Reply:
Ok, I'll remove it.